tests/zfs-tests/include/libtest.shlib

```diff
@@ -3386,6 +3386,18 @@ function md5digest
 	esac
 }
 
+#
+# Compare the MD5 digest of two files.
+#
+function cmp_md5s {
+	typeset file1=$1
+	typeset file2=$2
+
+	typeset sum1=$(md5digest $file1)
+	typeset sum2=$(md5digest $file2)
+	test "$sum1" = "$sum2"
+}
```

Comment:
Could you replace all `cmp_md5s` calls with `diff -s`? I know `diff -s` is supported even on Centos 7, but I don't know about FreeBSD

Reply:
Not sure why you'd even want to spec `-s`. `cmp_md5s a b` appears to be, very literally, `cmp a b >/dev/null`, but worse because it uses MD5(?!?!?!) and ignores the output; a better implementation would be 
```
function cmp
{
  command cmp "$@" >&2
}
```
to interact better with `log_must`, or, indeed, to just run `cmp`.

Reply:
Heh I've been using `diff -s` all these years and never knew about `cmp` :smile: 

I noticed later on that you were just moving `cmp_md5s` from a different place, and not adding it as new code, so I think it's ok to use it as you have it.  And then someday in a different PR, someone can go though and do a `cmp_md5s` - > `cmp` conversion.